### ISSUE TITLE
Moving the multiband LS upstream

### ISSUE THREAD

**Issue by bsipocz** [astroML/gatspy#38]
If I understand the status correctly, it's only the multiband LS that is unique in gatspy, the rest had been moved upstream to astropy a couple of years ago.

@jakevdp - how would you feel to move the multi band as well and maybe archive this repo? It would make somewhat  easier to communicate to the community that all your astroML & gatspy implementations has been moved, and receive the maintenance and occasional fixes and improvements in astropy.timeseries rather than at multiple places?

I volunteer to do the actual moving bit, but it's one of those cases where it feels better to ask for permission and opinion first rather than forgiveness after the fact.

**Comment by jakevdp** [comment]
Sure, if the astropy community thinks the multiband code belongs upstream, I'd be fine with that.

**Comment by rhiannonlynne** [comment]
Is this work underway? I would certainly use it, and I see that some of the community are considering opening a new work "NewGatspy" to continue updates/work on gatspy with needs ramping up as part of the LSST.

**Comment by bsipocz** [comment]
Yes, the LINCC group is doing some of this upstreaming. PR is here: https://github.com/astropy/astropy/pull/14016

**Comment by bsipocz** [comment]
@dougbrn, could you comment on whether there is anything else left in this library, once the PR from above is merged? If not, I would propose to move all the open issues upstream, and archive the repo for now.

**Comment by dougbrn** [comment]
Yeah, the multiband lombscargle going into astropy has everything from gatspy's multiband lombscargle. As far as what is left in the gatspy overall, we didn't include some of the other multiband gatspy functions in the port, such as the [Multiband SuperSmoother](http://www.astroml.org/gatspy/periodic/supersmoother.html#multiband-supersmoother) or the [Multiband Template Fitter](http://www.astroml.org/gatspy/periodic/template.html#multiband-template-fitting). So unless those were ported in other work, gatspy may still be the only home for them.

**Comment by bsipocz** [comment]
It would be lovely to see them being ported over, too, so everything is at one place.

**Comment by bsipocz** [comment]
I suppose as all stuff gets moved out from here, we should move the `datasets/fetch_...` functions upstream to [astroml/astroml?](https://github.com/astroML/astroML)

**Comment by bsipocz** [comment]
Given the comment https://github.com/astroML/gatspy/issues/38#issuecomment-1492697461 above, I'm closing this issue now. The moving of the supersmoother upstream has the astropy issue already, there is no need for multiple open issues for the same thing then.
